JSONAPI/Json/ResourceCollectionDocumentFormatter.cs

```diff
@@ -95,11 +101,7 @@ public Task Serialize(IResourceCollectionDocument document, JsonWriter writer)
                 writer.WriteEndArray();
             }
 
-            if (document.Metadata != null)
-            {
-                writer.WritePropertyName(MetaKeyName);
-                MetadataFormatter.Serialize(document.Metadata, writer);
-            }
+
```

Comment:
No it's not functionally necessary. But the specs and most samples start with the top-level metadata. I haven't seen any constarint which would enforce it.
Should I move it back?